### PR TITLE
DDF-1775 Updated parts of the DDF documentation to remove old references, duplicate documentation, and old configurations

### PR DESCRIPTION
**Pull request by stustison**
@codice/ddf-committers

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/434" target="_blank">ReviewNinja

**Comment by stustison** [comment]
Just a note. I'm pretty sure that I didn't catch everything. The documentation needs a HUGE overhaul to go through and update stuff. This was just my attempt to at least get the documentation to a point where it is not so incredibly wrong that someone who has never seen DDF wouldn't be able to set it up successfully.

File: distribution/docs/src/main/resources/ExtContents.adoc [hunk 1]
```diff
@@ -3217,7 +2416,7 @@ It is a jar file that contains a feature descriptor file and one or more OSGi bu
 The feature descriptor file identifies the application's name, the set of bundles that need to be installed, and any dependencies on other features that may need to be installed.
 
 ==== Creating a KAR File
-The recommended method for creating a KAR file is to use the `features-maven-plugin`, which has a `create-kar` goal (available as of Karaf v2.2.5, which {branding} 2.X is based upon).
+The recommended method for creating a KAR file is to use the `features-maven-plugin`, which has a `create-kar` goal.
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/ExtContents.adoc]: yeah we are well beyond needing to reference older Karaf versions

**Comment by shaundmorris** [comment]
nice updates :+1:

**Pull request merged**